src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain
+
+class Lotto(numbers: List<Int>) {
+    private val _lottoNumbers: List<LottoNumber> = numbers.map { LottoNumber(it) }
+    val lottoNumbers: List<LottoNumber>
+        get() = _lottoNumbers.toList()
+
+
+    init {
+        require(numbers.size == SIZE_OF_LOTTO_NUMBERS) { "The number of lotto numbers must be $SIZE_OF_LOTTO_NUMBERS." }
+        require(numbers.distinct().size == SIZE_OF_LOTTO_NUMBERS) { "Lotto numbers must be unique." }
+    }
```

Comment:
```suggestion
    init {
        require(numbers.size == SIZE_OF_LOTTO_NUMBERS) { "The number of lotto numbers must be $SIZE_OF_LOTTO_NUMBERS." }
        require(numbers.distinct().size == numbers.size) { "Lotto numbers must be unique." }
    }
```

might be more precise

Reply:
agreed